src/klein/_resource.py

```diff
@@ -19,7 +20,18 @@
 
 
 if TYPE_CHECKING:
-    from ._app import ErrorHandlers, Klein, KleinRenderable
+    # NB: circular import, must not be imported at runtime.
+    from ._app import (
+        ErrorHandlers,
+        Klein,
+        KleinRenderable,
+        KleinRouteHandler,
+        RouteMetadata,
+    )
+
+
+def route_metadata(handler: KleinRouteHandler) -> RouteMetadata:
+    return handler  # type:ignore[return-value]
```

Comment:
Is a `cast` aesthetically better than a type:ignore here?

Reply:
I tend to prefer the `type:ignore` for helpers like this because it doesn't add a *second* useless runtime method call.